src/game.cpp

```diff
@@ -133,6 +134,9 @@ static void Init() {
     TipsNotification::Init();
     DelusionTrigger::Init();
     UI::MapSystem::Init();
+    if (Profile::Vm::GameInstructionSet == +Vm::InstructionSet::CC) {
```

Comment:
I guess it's not a problem right now, but regular C;C does not have this, does it?

Reply:
It doesn't

Reply:
Init does already check if LUA file contains the YesNoTrigger table push, so it's fine to remove